canvas_modules/common-canvas/src/common-properties/controls/toggle/toggle.scss

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2017-2022 Elyra Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.properties-toggle {
+	margin-bottom: 14px;
```

Comment:
Why is this 14px? If we need a margin this should be a token.

Reply:
used token.

Reply:
What is this for?

Reply:
Used it to center toggles in tables , isnt the right approach?

without token
<img width="598" alt="image" src="https://user-images.githubusercontent.com/89609176/187508584-b8c0d3b0-c95b-46b2-ae78-21d6155dcb25.png">
with token
<img width="598" alt="image" src="https://user-images.githubusercontent.com/89609176/187508667-c3540276-0860-4862-89ce-238fefebaec3.png">

Reply:
No it adds too much space to the toggle when not in tables.  

When in a table you should override `bx--toggle__switch` and set the margin to 0.  This is only for tables though.  Also make sure to add a comment.
![Screen Shot 2022-08-30 at 1 03 55 PM](https://user-images.githubusercontent.com/21126000/187510571-2d6c45e4-2475-462a-afcc-13daae560755.png)

Reply:
Looks better.  Can you use this
```
.properties-table-cell-control { // overrides when in a table cell
	.properties-toggle {
		.bx--toggle__switch { // override carbon when inside a table
			margin: $spacing-02 0;
		}
	}
}
```